code401challenges/src/test/java/code401challenges/stacksandqueues/QueueTest.java

```diff
@@ -0,0 +1,51 @@
+package code401challenges.stacksandqueues;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class QueueTest {
+    @Test
+    public void testEnqueue() {
+        Queue queue = new Queue();
```

Comment:
Since you made a generic Queue, when you instantiate a new Queue use the diamond to declare the type, like: Queue<Integer> queue = new Queue<>();